src/ddspdrum/module.py

```diff
@@ -411,17 +500,50 @@ def __init__(
         self.noise_module = noise_module
         self.vca = vca
 
+        # Setup parameters for this drum synth
+        self.add_parameter("sustain_duration", sustain_duration, 0, 60, scale=0.5)
+        self.add_parameter("vco_1_ratio", vco_1_ratio, 0, 1)
```

Comment:
I know it's confusing, sorry. I'd say each Drum has two "ratios":

(1) noise_ratio --> amount of noise.
(2) vco_ratio --> mix between vco 0/1. this is equivalent to a mix knob.

It might be confusing to call it "vco_1_ratio" when really it affects the level of both oscillators.

Reply:
Sounds good